benchmark/benchmarks.js

```diff
@@ -48,6 +48,10 @@ export default function makeBenchmarks() {
         h3core.h3ToGeoBoundary(h3Index);
     });
 
+    suite.add('h3GetFaces', () => {
```

Comment:
I do wonder about the validity of the benchmarks when the h3Index value does not change. V8 might hyper optimize some of the code, like the size of the return array, that wouldn't happen with a blend of indices including edge-crossing hexagons and pentagons. Not a big deal, though.

Reply:
That's valid, but really just points to the difficulty of benchmarking JS to begin with :). I'm not aware of any V8 optimizations for repeated function calls with the same input, but it's possible 🤷‍♂. However, I think from my perspective, the main goals of these benchmarks are:

- Get a sense of the order of magnitude
- Check for regressions or improvements when making relevant changes

and I think simple benchmarks are probably sufficient for this.

Reply:
I was referring to V8's optimizations on input and output types. It perhaps could be turning the arbitrary output array into an unboxed string internally.